### PR TITLE
Changes in  bundles to upgrade to php lvl 3

### PR DESCRIPTION
**Pull request by anxo188**
There are some dirty comits and some errors remaining

### CODE REVIEW

File: doc/docker/pumukit/docker-entrypoint.sh [hunk 1]
```diff
@@ -19,6 +19,7 @@ if [ "$1" = 'php-fpm' ] || [ "$1" = 'bin/console' ]; then
 	bin/console pumukit:init:repo all --force
     fi
 fi
-php bin/console fos:user:create $PUMUKIT_USER $PUMUKIT_MAIL $PUMUKIT_PASS
-
+set +e
+php bin/console fos:user:create $PUMUKIT_USER $PUMUKIT_MAIL $PUMUKIT_PASS || true
```
Comment (rubenrua) [inline doc/docker/pumukit/docker-entrypoint.sh]: Add `--super-admin`

File: src/Pumukit/EncoderBundle/Document/Job.php [hunk 2]
```diff
@@ -765,7 +765,7 @@ public function getOutput()
      */
     public function setInitVars($initVars)
     {
-        $this->initVars = $initVars;
+        array_push($this->initVars,$initVars);
```
Comment (rubenrua) [inline src/Pumukit/EncoderBundle/Document/Job.php]: keep `$this->initVars = $initVars;` and change PHPDoc to `@param array $initVars`

File: src/Pumukit/EncoderBundle/Services/PicService.php [hunk 3]
```diff
@@ -87,7 +87,7 @@ public function findPicsByOptions($id = null, $size = null, $path = null, $exten
             array_push($pipeline, $match);
         }
 
-        if ($extension) {
+        if (isset($extension)) {
```
Comment (rubenrua) [inline src/Pumukit/EncoderBundle/Services/PicService.php]: why?

File: doc/docker/pumukit/docker-entrypoint.sh (hunk 1 above)
Comment (anxo188) [reply doc/docker/pumukit/docker-entrypoint.sh]: changed in the previous pull request, on merge its expected to generate a conflict to solve

File: src/Pumukit/EncoderBundle/Document/Job.php (hunk 2 above)
Comment (anxo188) [reply src/Pumukit/EncoderBundle/Document/Job.php]: changed in the previous pull request, on merge its expected to generate a conflict to solve

**Pull request closed without merging**